sources/packages/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -1,4 +1,4 @@
-export interface ApplicationDetails {
+export class ActiveApplicationDataAPIOutDTO {
```

Comment:
These models are exclusively used by the location controllers therefore I would suggest moving them to the scope that they belong.